common/djangoapps/terrain/stubs/lti.py

```diff
@@ -266,7 +266,7 @@ def _oauth_sign(self, url, body, content_type=u'application/x-www-form-urlencode
 
         # Calculate and encode body hash. See http://oauth.googlecode.com/svn/spec/ext/body_hash/1.0/oauth-bodyhash.html
         sha1 = hashlib.sha1()
-        sha1.update(body)
+        sha1.update(body if six.PY2 else body.encode('utf-8'))
```

Comment:
If body is a string or unicode object in py2, you should be able to call the encode function on it directly.

I would expect that you can just say `body.encode('utf-8')` here and it would work in both python2 and 3 because python2 will allow you to call encode on a byte string.